core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/components/edit-ema-palette/components/edit-ema-palette-content-type/edit-ema-palette-content-type.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="content-search">
+    <span class="p-input-icon-right">
+        <i class="pi pi-search"></i>
+        <input
+            [placeholder]="'Search' | dm"
+            [formControl]="control"
+            [value]="filter"
+            type="search"
+            pInputText />
+    </span>
+</div>
+
+<div class="content-types">
+    <dot-spinner
+        *ngIf="paletteStatus === EDIT_EMA_PALETTE_STATUS.LOADING; else content"
+        [size]="'40px'"
+        [borderSize]="'8px'"></dot-spinner>
+
+    <ng-template #content>
+        <div
+            class="content-type-card"
+            *ngFor="let contenttype of contentTypes"
+            [attr.data-item]="{ variable: contenttype.variable, name: contenttype.name } | json"
+            (dragstart)="dragStart.emit($event)"
+            (dragend)="dragEnd.emit($event)"
+            data-type="contentlet"
+            draggable="true">
+            <i
+                class="pi pi-angle-right icon-go-content"
+                (click)="showContentletsFromContentType(contenttype.name)"></i>
+
+            <div class="content-type-card__content">
+                <dot-icon [name]="contenttype.icon"></dot-icon>
+                <span>
+                    {{ contenttype.name }}
+                </span>
+            </div>
+            <dot-icon class="drag-icon" name="drag_indicator"></dot-icon>
```

Comment:
Primengs

Reply:
Primeng dont have that icon https://www.primefaces.org/primeng-v15-lts/icons#list